_docs/faq.md

```diff
@@ -217,6 +217,26 @@ FAQ tags: 
 
 Back in 1998, the Gene Ontology began as a collaboration between three model organism databases, namely Flybase (Drosophila), the Saccharomyces Genome Database (SGD), and the Mouse Genome Database (MGD). Today, the GO Consortium is formed by many databases, including several of the world's major repositories for plant, animal, and microbial genomes. Visit [this page to see a complete list of member organizations](/docs/annotation-contributors/) of the Gene Ontology Consortium.
 
+---------------------------------------------------------------------------
+## What is the difference between GO, GOC, GO Central, and GOA?
+{::comment}
+
+<span class="rdf-meta element-hidden" property="dc:title" content="What is the difference between GO, GOC, GO Central, and GOA?"></span>
+FAQ tags: 
+
+[consortium](/faq-tags/consortium)
+{:/comment}
+
+GO, GOC, GO Central, and GOA are similar but not quite interchangable. Here's a quick breakdown of these commonly confused acronyms:
+#### GO
+- **G**ene **O**ntology: Although formally we use GO to refer to the entire GO knowledgebase, this can also refer to just the ontology. As well as the [ontology](http://geneontology.org/docs/ontology-documentation/) and [annotations](http://geneontology.org/docs/go-annotations/), GO's products include the [AmiGO browser](http://amigo.geneontology.org/amigo), [GO-Causal Activity Models (GO-CAMs)](http://geneontology.org/go-cam), [GO Term Matrix](geneontology.org/docs/tools-overview#explore-co-annotations-with-the-go-term-matrix), etc.
+#### GOC 
+-   **G**ene **O**ntology **C**onsortium: The databases, labs, and other groups that regularly contribute annotations, shape the ontology, or are otherwise active in the maintenance of the GO knowledgebase. [See a complete list of member organizations](/docs/annotation-contributors/) of the Gene Ontology Consortium.
+#### GO Central
+-   The GO Central project is embedded in the larger [GO Consortium](http://geneontology.org/docs/whoweare/) and circle of [GO Collaborations](http://geneontology.org/docs/collaborations/). Within this complex landscape of resources, GO Central has several activities, including ontology development, tool development, annotation support, quality assurance, and outreach. 
```

Comment:
@pgaudet I think Chris had a Venn diagram of all these that he showed at a meeting? Isn't Central also defined by funding?

Reply:
@kltm, pascale found the GO_central description from a grant, not sure which one but it could be roughly dated as it referred to the "newly formed Alliance for Genome Resources"

Reply:
Fair enough. I dropped the slide I think I was thinking of in the slack. I'm not sure if it actually helps here though.

Reply:
I actually like that image, a day may come when I tidy it up and put it on the site, but it is not this day.